utils/installer/install.sh

```diff
@@ -182,7 +182,7 @@ installextrapackages() {
 # Welcome
 echo 'Installing LunarVim'
 
-if [[ $* == *--overwrite* ]]; then
+if [ -n "$(echo "$@" | grep "\-\-overwrite")" ]; then
```

Comment:
Nice solution, but there is an even better way, [parameters expansion](https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_02).
Check [this](https://stackoverflow.com/questions/2829613/how-do-you-tell-if-a-string-contains-another-string-in-posix-sh) post for example

Reply:
I like it but for some reason it didn't work using '!=' sign in dash on $@.
I'm using the case solution though - it is more extendable. thanks.